sites/walmart.py

```diff
@@ -468,4 +468,11 @@ def handle_captcha(self, url_to_open):
                 self.session.cookies.set(c['name'], c['value'], path=c['path'], domain=c['domain'])
 
     def is_captcha(self, text):
-        return "captchajs" in text or "captcha.js" in text
\ No newline at end of file
+        return "captchajs" in text or "captcha.js" in text
+
+    def random_delay(self, delay, start, stop):
```

Comment:
Can we move this utils.py? Seems like a simple useful function that could be used elsewhere

Reply:
Sounds good, I'll move it and update it later today.

Reply:
LGTM